core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -142,6 +142,7 @@ private static void onResponse(
         final int childrenSize = log.children().size();
         if (childrenSize > 0) {
             metrics.actualRequests().increment(childrenSize);
+            metrics.retryingRequests().record(childrenSize);
```

Comment:
`childrenSize` is 1 even if a request is completed without retrying when `RetryingClient` is configured. 1 `retries` sounds like one retry was made and the second request succeeded. What do you think of subtracting 1 from `childrenSize`?
```suggestion
            metrics.retryingRequests().record(childrenSize - 1);
```

Reply:
Thanks for your suggestion!
I understood that `retries` is total retries were executed no matter they succeeded or failed. 
For example: If I make 3 retrying requests, the first two failed and the last one succeeded. We will count only the failed retries and in this case `retries=2`?

Reply:
SGTM

Reply:
I was rather thinking (2) and (3) should both yield `retries=2`, just with different tags (e.g. `status=success` vs `status=failure`)